src/ui/ToggleEditor.tsx

```diff
@@ -206,19 +190,19 @@ class ToggleEditor extends Component<ToggleEditorProps, ToggleEditorState> {
     SHARED.announcer = announcements;
 
     let defaultOptions = {
-      parse: this.parse,
+      parse: this.props.language.parse,
       incrementalRendering: true,
       collapseAll: true
     };
     this.options = {...defaultOptions, ...props.options};
-    this.hasMounted = false;
     SHARED.recordedMarks = new Map();
     this.eventHandlers = {}; // blank event-handler record
 
     // make sure 'this' always refers to ToggleEditor
     // see https://reactjs.org/docs/handling-events.html
     this.showDialog  = this.showDialog.bind(this);
     this.closeDialog = this.closeDialog.bind(this);
```

Comment:
These lines could be removed by just making `showDialog` and `closeDialog` into arrow functions in the same way as `handleToggle`

Reply:
Fixed.